run_clover.py

```diff
@@ -0,0 +1,102 @@
+from montecarlo.node import Node
```

Comment:
run_clover shouldn't duplicate the run, instead, it should be parametric in the run like experiments, maybe?

Reply:
Going to tackle in separate PR